mdpow/templates/bar_opls.mdp

```diff
@@ -40,16 +44,20 @@ ns_type                  = grid
 pbc                      = xyz
 periodic_molecules       = no
 ; nblist cut-off
+; WARNING: GROMACS <2021 incorrectly deals with exclusions. See https://gitlab.com/gromacs/gromacs/-/issues/3403
+;          GROMACS >= 2021 requires rlist >= diameter of solute for couple-intramol = no FEP calculations
+;                          In order to set rlist to a fixed value, disable the automatic verlet buffer with -1.
+;                          by UNCOMMENTING THE NEXT LINE and INCREASE rlist AS NECESSARY.
+;verlet-buffer-tolerance  = -1
 rlist                    = 1.0
+cutoff-scheme            = Verlet
```

Comment:
Adding this line probably makes the 4.6.5 FEP fail.

Reply:
I could probably leave it out (or comment it out) and let Gromacs choose the scheme.

Reply:
Solved in 9b275cb by running the 4.6.5 test with the group scheme + added note to CHANGES.